Gemfile

```diff
@@ -0,0 +1,3 @@
+source "https://rubygems.org"
```

Comment:
I've removed that particular line as part of a follow up commit.
I was thinking though, did you mean just that line or adding Gemfile + bundler in general?